packages/date-time/src/components/CalendarDayGrid/CalendarDayGrid.test.tsx

```diff
@@ -0,0 +1,28 @@
+import { CalendarDayGrid } from './CalendarDayGrid';
+import { defaultDayPickerStrings, DateRangeType, DayOfWeek, FirstWeekOfYear, ICalendarStrings } from '../Calendar';
+import { isConformant } from '../../common/isConformant';
+
+describe('CalendarDayGrid', () => {
+  const timeFormatter = {
+    formatMonthDayYear: (date: Date, strings?: ICalendarStrings) => 'm/d/y',
+    formatMonthYear: (date: Date, strings?: ICalendarStrings) => 'm/y',
+    formatDay: (date: Date) => 'd',
+    formatYear: (date: Date) => 'y',
+  };
+
+  isConformant({
+    Component: CalendarDayGrid,
+    displayName: 'CalendarDayGrid',
+    requiredProps: {
+      strings: defaultDayPickerStrings,
+      selectedDate: new Date(),
```

Comment:
This is fine for now but may become an issue later if we add snapshot tests to conformance